### PR TITLE
Fixes #2843 and introduce getAttributes() native method

### LINKED ISSUE #2843: [SessionSupport] getLastAccessedTime returns the current time
According to the implementation, "getLastAccessedTime" will always return the current time (as it is also the "last" access time). But, the user expects the previously last access time (last one before the current one).

### PR DESCRIPTION
**Pull request by chamil321**
- Fix #2843

- Introduce getAttributes native method to HTTP session API which returns attribute key-value pairs as a map. This is requested during Session API review discussion
```ballerina
native function <Session session> getAttributes () (map);
```

### CODE REVIEW

File: modules/ballerina-http/src/main/java/org/ballerinalang/net/http/session/HTTPSession.java [hunk 1]
```diff
@@ -58,22 +59,26 @@ public String getId() {
...
-        checkValidity();
         return attributeMap.keySet().toArray(new String[attributeMap.size()]);
     }
 
+    @Override
+    public BMap<String, BValue> getAttributes() {
+        if (!attributeMap.isEmpty()) {
+            return attributeMap;
+        }
+        return null;
```
Comment (SupunS) [inline modules/ballerina-http/src/main/java/org/ballerinalang/net/http/session/HTTPSession.java]: IMO we should always return the map, regardless its empty or not.

File: modules/ballerina-http/src/main/java/org/ballerinalang/net/http/session/Session.java [hunk 2]
```diff
@@ -82,11 +83,18 @@
     /**
      * Get attribute name list.
      *
-     * @return name array.
+     * @return array of keys.
```
Comment (SupunS) [inline modules/ballerina-http/src/main/java/org/ballerinalang/net/http/session/Session.java]: shouldn't it be "array of attribute names"?

File: modules/ballerina-http/src/main/java/org/ballerinalang/net/http/session/HTTPSession.java (hunk 1 above)
Comment (chamil321) [reply modules/ballerina-http/src/main/java/org/ballerinalang/net/http/session/HTTPSession.java]: Ack

File: modules/ballerina-http/src/main/java/org/ballerinalang/net/http/session/Session.java (hunk 2 above)
Comment (chamil321) [reply modules/ballerina-http/src/main/java/org/ballerinalang/net/http/session/Session.java]: Ack

**Pull request merged**